aten/src/ATen/native/native_functions.yaml

```diff
@@ -5018,6 +5018,24 @@
   device_check: NoCheck
   device_guard: False
 
+- func: sym_size.int(Tensor self, int dim) -> SymInt
+  variants: function
+  device_check: NoCheck
+  device_guard: False
+  tags: canonical
+
+- func: sym_numel(Tensor self) -> SymInt
+  variants: function
+  device_check: NoCheck
+  device_guard: False
+  tags: canonical
+
+- func: sym_storage_offset(Tensor self) -> SymInt
+  variants: function
+  device_check: NoCheck
+  device_guard: False
+  # tags: canonical   # I am not sure about how this op is used
```

Comment:
Do we have use of this op in real model ? 
I see it's only used in tensor's symbolic metadata.

Reply:
I don't think we've ever seen it but in principle it is necessary